tests/contrib/asyncpg/py35/test_asyncpg.py

```diff
@@ -0,0 +1,231 @@
+# flake8: noqa
```

Comment:
I have a feeling they copied this to a new place, so need to find the equivalent